client/components/cloud/project/billing/additional-services/additional-services.component.controller.js

```diff
@@ -0,0 +1,90 @@
+class AdditionalServicesComponentCtrl {
+  constructor($stateParams, $translate, CucControllerHelper,
+    OvhApiMe, CucServiceHelper, OvhApiCloudProjectInstance) {
+    this.$stateParams = $stateParams;
+    this.$translate = $translate;
+    this.CucControllerHelper = CucControllerHelper;
+    this.OvhApiMe = OvhApiMe;
+    this.CucServiceHelper = CucServiceHelper;
+    this.OvhApiCloudProjectInstance = OvhApiCloudProjectInstance;
+  }
+
+  $onInit() {
+    this.initAccordians();
+    if (this.getPlatformCount() > 0) {
+      this.initInstances();
+    }
+  }
+
+  /**
+   * initiate UI accordian state, one accordian per platform
+   */
+  initAccordians() {
+    const accordions = {};
+    _.forEach(_.keys(this.services), (key) => {
```

Comment:
Could `_.mapValues` work here 
```suggestion
  const accordions =   _.mapValues(this.services, () => false);
```